packages/plugin-core/src/_extension.ts

```diff
@@ -65,16 +63,14 @@ async function reloadWorkspace() {
     return maybeEngine;
   }
   Logger.info({ ctx, msg: "post-ws.reloadWorkspace" });
-  // check if first time install workspace, if so, show tutorial
```

Comment:
can you pull from master? the logic for this has changed so we'll want to make sure this is still compatible with how we detect initial installs